src/libs/blueprint/conduit_blueprint_mpi_mesh.hpp

```diff
@@ -76,6 +76,68 @@ void CONDUIT_BLUEPRINT_API generate_domain_to_rank_map(
 index_t CONDUIT_BLUEPRINT_API number_of_domains(const conduit::Node &mesh,
                                                 MPI_Comm comm);
 
+//-------------------------------------------------------------------------
+void CONDUIT_BLUEPRINT_API delegate(const conduit::Node &mesh,
```

Comment:
To help folks down the line + since its part of the interface - can you add a comment about what delegate is used for?

Reply:
Yes, definitely. I think a better name would help as well; the fact that "delegate" can be a noun and a verb makes this very confusing.